packages/deployment/upgrade-test/upgrade-test-scripts/unreleased-upgrade/pre.test.js

```diff
@@ -19,14 +19,3 @@ test('Ensure Network Vat is at 0', async t => {
   const incarnation = await getIncarnation('network');
   t.is(incarnation, 0);
 });
-
-test('Ensure MaxBytes param was updated', async t => {
```

Comment:
good time to point out the other tests could change. e.g. when we upgrade Zoe, then line 15 will have to have `1`